package/scripts/postinstall

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Would be good to ensure `brew style` checks these by adding to https://github.com/Homebrew/brew/blob/f679990d7918d9f99d9c6c99b7bec504bc4088fe/Library/Homebrew/style.rb#L260-L276